docs/designers-developers/developers/tutorials/block-tutorial/generate-blocks-with-wp-cli.md

```diff
@@ -62,7 +62,7 @@ This will generate 4 files inside the `movies` plugin directory. All files conta
  * Registers all block assets so that they can be enqueued through Gutenberg in
  * the corresponding context.
  *
- * @see https://wordpress.org/gutenberg/handbook/blocks/writing-your-first-block-type/#enqueuing-block-scripts
+ * @see https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/block-tutorial/writing-your-first-block-type/
```

Comment:
We should backport those changes to WP-CLI, too :)

Reply:
Done :) https://github.com/wp-cli/scaffold-command/pull/199